source/features/link-to-requested-change-comment.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import onetime from 'onetime';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import {wrap} from '../helpers/dom-utils';
+import features from '.';
+
+function init(): void {
+	observe('.review-item .dropdown-item[href^="#pullrequestreview-"]', {
+		constructor: HTMLAnchorElement,
+		add(messageContainer) {
+			const element = select('.review-status-item div[title*="requested changes"]')?.lastChild;
+
+			if (element) {
+				wrap(element, <a href={messageContainer.href}/>);
+			}
```

Comment:
Conditions like this makes any future errors silent, however this element is expected to be "not found" when GitHub starts hiding some comments on long discussions.

It's not ideal, but we don't have a reasonable solution here.